src/components/WorkOrders/JobView.js

```diff
@@ -6,15 +6,16 @@ import ErrorMessage from '../Errors/ErrorMessage/ErrorMessage'
 import { sortObjectsByDateKey } from '../../utils/date'
 
 const JobView = () => {
+  const [pageNumber, setPageNumber] = useState(1)
```

Comment:
Should we not have page number within the url so it's easily shareable and a user can track what page they are on ? At the moment the url stays constant right so if someone would refresh the page they would just start off at page 1 again

Reply:
Think you can use Router here perhaps